utilities/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ file(GLOB sources *.cc)
 file(GLOB headers *.h)
 
 add_library(utilities SHARED ${sources} ${headers} )
-target_link_libraries(utilities datamodel podio ${ROOT_LIBRARIES} ${ROOT_COMPONENT_LIBRARIES})
+target_link_libraries(utilities datamodel podio ROOT::Physics ${ROOT_COMPONENT_LIBRARIES})
```

Comment:
What is `${ROOT_COMPONENT_LIBRARIES}` here actually? Can we remove it and add Root targets also?

Reply:
It's indeed empty, so I removed it.